Complete op throws exception in *.clj file with 0.22.0-beta10

## Expected behavior

Returns completion candidates.

## Actual behavior

```
foo.core=> ERROR: Unhandled REPL handler exception processing message {:context (__prefix__), :extra-metadata [arglists doc], :session 90fab402-ec8a-48c8-87b2-944fccc6165c, :ns foo.core, :symbol foo, :op complete}
Syntax error compiling at (cider/nrepl/inlined_deps/suitable/v0v2v1/suitable/complete_for_nrepl.clj:1:1).
        at clojure.lang.Compiler.load(Compiler.java:7647)                         
        at clojure.lang.RT.loadResourceScript(RT.java:381)                  
        at clojure.lang.RT.loadResourceScript(RT.java:372)                     
        ...
Caused by: java.io.FileNotFoundException: Could not locate cljs/repl__init.class, cljs/repl.clj or cljs/repl.cljc on classpath.
        at clojure.lang.RT.load(RT.java:466)                              
        at clojure.lang.RT.load(RT.java:428)                                      
        at clojure.core$load$fn__6824.invoke(core.clj:6126)   
        ...
```

It seems that  requireing `cljs.repl` directly in clj-suitable is the cause . (cc @rksm )
https://github.com/rksm/clj-suitable/blob/0.2.1/src/main/suitable/complete_for_nrepl.clj#L4

## Steps to reproduce the problem

* `lein new foo && cd foo`
* Add `:plugins [[cider/cider-nrepl "0.22.0-beta10"]]` to project.clj
* `lein repl`
* Open editor, connect to nREPL, do completion

## Environment & Version information

### cider-nrepl version

0.22.0-beta10

### Java version

openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1ubuntu1~18.04.1-b10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)

### Operating system

Ubuntu 18.04

--- a/project.clj
+++ b/project.clj
@@ -11,7 +11,7 @@
                  ^:inline-dep [fipp "0.6.18"] ; can be removed in unresolved-tree mode
                  ^:inline-dep [compliment "0.3.9"]
                  ^:inline-dep [cljs-tooling "0.3.1"]
-                 ^:inline-dep [org.rksm/suitable "0.2.1" :exclusions [org.clojure/clojurescript]]
+                 ^:inline-dep [org.rksm/suitable "0.2.4" :exclusions [org.clojure/clojurescript]]
                  ^:inline-dep [cljfmt "0.6.4" :exclusions [org.clojure/clojurescript]]
                  ^:inline-dep [org.clojure/tools.namespace "0.3.1"]
                  ^:inline-dep [org.clojure/tools.trace "0.7.10"]

